rules/js/react/.esmintrc

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Is this file supposed to be `.eslintrc`?

Reply:
Haha yesh